src/main/java/com/artipie/RepoPermissions.java

```diff
@@ -66,12 +67,17 @@
     CompletionStage<Void> remove(String repo);
 
     /**
-     * Adds or updates repository permission.
+     * Adds or updates repository permissions.
      * @param repo Repository name
      * @param permissions Permissions list
+     * @param patterns Included path patterns
      * @return Completion action
      */
-    CompletionStage<Void> addUpdate(String repo, Collection<UserPermission> permissions);
+    CompletionStage<Void> update(
+        String repo,
+        Collection<UserPermission> permissions,
+        List<String> patterns
```

Comment:
@olegmoz I'd suggest use `Collection` here instead of list

Reply:
@olenagerasimova thanks, you are right. Fixed it